LibGit2Sharp/SubmoduleCollection.cs

```
@@ -49,6 +49,70 @@ internal SubmoduleCollection(Repository repo)
         }
 
         /// <summary>
+        /// Initialize specified submodule.
+        /// </summary>
+        /// <param name="name">The name of the submodule to update.</param>
+        /// <param name="overwrite">Overwrite existing entries.</param>
+        public virtual void Init(string name, bool overwrite)
```

Could you please describe what happens when Initializing over a existing one (and passing overwrite = false)? Throws/silently swallows....